core/src/main/java/com/sequenceiq/cloudbreak/reactor/handler/cluster/dr/BackupRestoreSaltConfigGenerator.java

```diff
@@ -34,7 +34,6 @@ public SaltConfig createSaltConfig(String location, String backupId, Stack stack
 
         Map<String, String> disasterRecoveryValues = new HashMap<>();
         disasterRecoveryValues.put(OBJECT_STORAGE_URL_KEY, fullLocation);
-        disasterRecoveryValues.put(AWS_REGION_KEY, stack.getRegion());
```

Comment:
Why is the region removed?

Reply:
We got the suggestion to use the `hdfs` cli instead of s3 to upload/download.

It was my understanding that we don't have to specify the region in that case, the hdfs configurations should take care of things?

Reply:
@keyki  AWS CLI would depend on permissions granted to the instance profile of the datalake node which is currently LOG_ROLE. Grating more permissions to LOG_ROLE was not a good idea.
With this change, we will be using hdfs client to write to S3. This uses S3a. All the services which to S3, internally use S3a. With the configuration we are using, S3a fetches the region information of the bucket and then constructs the request accordingly.

Reply:
I'm resolving this conversation, given that we have a reason and there's been no further concern.